json/targeting.json

```diff
@@ -133,6 +133,22 @@
         }
       }
     },
+    "ifRule": {
```

Comment:
FYI @austindrenski , the JSON is generated from the YAML, and the CI makes sure they are in sync.